### PR TITLE
Docs: Fixed the ordering of releases in releases page

### LINKED ISSUE #2126: [Docs] Ordering of release versions on Releases page
@SudhanAruna, well done! One small item to deal with here: the descending order of releases by version number is being itemized slightly out of order with the string “v0.4.9” coming before “v0.4.25", because the “9” is greater than “2", as opposed to “25” being greater than “9".

_Originally posted by @leecalcote in https://github.com/layer5io/meshery/issues/2039#issuecomment-749192526_

### PR DESCRIPTION
**Pull request by SudhanAruna**
Signed-off-by: Arunagirinadan Sudharshan <sudhan.aruna.97@gmail.com>

**Description**
This PR fixes the ordering of releases in release page and modified release-drafter workflow file to add 'date' front matter in future automated release note .md files.

This PR fixes #2126

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Hello! :)

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
         id: md_writer
         if: github.event_name != 'pull_request' && startsWith(github.ref, 'refs/tags/') && success()
         run: |
-          printf '%b\n' "---\nlayout: default\n---\n\n${{ steps.release_drafter.outputs.body }} " > ./docs/_releases/${{ steps.release_drafter.outputs.tag_name }}.md
+          printf '%b\n' "---\nlayout: default\ndate: $(date +'%d-%m-%Y')\n---\n\n${{ steps.release_drafter.outputs.body }} " > ./docs/_releases/${{ steps.release_drafter.outputs.tag_name }}.md
```
Comment (leecalcote) [inline .github/workflows/release-drafter.yml:30]:
```suggestion
          printf '%b\n' "---\nlayout: default\ndate: $(date +'%m-%d-%Y')\n---\n\n${{ steps.release_drafter.outputs.body }} " > ./docs/_releases/${{ steps.release_drafter.outputs.tag_name }}.md
```

Comment (SudhanAruna) [reply .github/workflows/release-drafter.yml]: We should change the date format in previous release file as well (currently it is "%d-%m-%Y")

**Review by leecalcote — COMMENTED** [review]
Looking good. 👍

**Comment by SudhanAruna** [comment]
@leecalcote Thank you for completing it.

**Pull request merged**